### DISCUSSION TITLE
AttributeError: module 'glm' has no attribute 'vec3'

### DISCUSSION THREAD

**Discussion by AlkisPis** [community/community#129250] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Product Feedback

### Body

In 3D-Graphics-Engine (https://github.com/StanislavPetrovV/3D-Graphics-Engine) and specifically in the module light.py (https://github.com/StanislavPetrovV/3D-Graphics-Engine/blame/main/light.py), the instruction `self.position = glm.vec3(position)` raises "AttributeError: module 'glm' has no attribute 'vec3'":  

    `class Light:
      def __init__(self, position=(50, 50, -10), color=(1, 1, 1)):
        self.position = glm.vec3(position) # This raises "AttributeError: module 'glm' has no attribute 'vec3'"
        self.color = glm.vec3(color) # AttributeError: module 'glm' has no attribute 'vec3'!!
        self.direction = glm.vec3(0, 0, 0)
        # intensities
        self.Ia = 0.06 * self.color  # ambient
        self.Id = 0.8 * self.color  # diffuse
        self.Is = 1.0 * self.color  # specular
        # view matrix
        self.m_view_light = self.get_view_matrix()`

How can this be overcome?

Versions:
Pytjon 3.12
gllm 0.4.4

**Comment by ebndev** [comment]
Hi @AlkisPis, thanks for being a part of the GitHub Community!

It looks like you may have ended up here by clicking “Send feedback” on a user’s profile, is that right? 

That link is meant for users to provide feedback for Achievements, which leads to the Profile category of GitHub Discussions, a public forum.

We are working on a solution to that however, you might want to try going back to the project you were seeking support for and open an issue in the project's repository or contact them on their website / social media. There is currently no way to contact users directly on GitHub.

We want to make sure you’re getting the best support you can and that your message goes where you intended, but this may not be the right place.

Best of luck!

**Comment by AlkisPis** [comment]
I'm sorry about that. There was no other way to report the problem. If you indeed are working on a solution to avoid this confusion, you can just add a **visible** button or link un the code page where one can report problems with the code.

**Comment by davidson16807** [comment]
I got this error when I accidentally called `import pyglm as glm` and not `from pyglm import glm` so I suspect somebody forgot which way to import glm correctly.
